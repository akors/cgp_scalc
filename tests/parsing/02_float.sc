11.1 + 22.11
9 / 11.9
0.000011 * 99227.11
21 / 78
