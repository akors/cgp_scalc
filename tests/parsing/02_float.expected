33.21
0.756303
1.0915
0.269231
